test/integration/targets/win_acl_inheritance/tasks/main.yml

```diff
@@ -0,0 +1,173 @@
+---
+ # Test setup
+- name: remove test folder for baseline
+  win_file:
+    path: "{{test_win_acl_inheritance_path}}"
+    state: absent
+
+- name: create test folders
+  win_file:
+    path: "{{test_win_acl_inheritance_path}}\\folder"
```

Comment:
We need to document this. Single quotes are definitely preferred, but there is still an issue with a trailing backslash :-(